### PR TITLE
chore[Breaking]: rename dev option to devMiddleware

### PR DESCRIPTION
**Pull request by anshumanv**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [x] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### For Bugs and Features; did you add new tests?
Existing

### Motivation / Use-Case
dev is misleading, rename to devMiddleware to convey intention

### Breaking Changes
Yes, `dev` renamed to `devMiddleware`

### CODE REVIEW

File: lib/options.json [hunk 1]
```diff
@@ -227,7 +227,7 @@
       "type": "boolean",
       "description": "Enable gzip compression for everything served. https://webpack.js.org/configuration/dev-server/#devservercompress"
     },
-    "dev": {
+    "devMiddleware": {
       "type": "object",
       "description": "The bundled files will be available in the browser under this path. https://webpack.js.org/configuration/dev-server/#devserverdev-"
```
Comment (alexander-akait) [inline lib/options.json:232]: Let's fix description, we provide options here for webpack-dev-middleware, which handle webpack assets

Comment (anshumanv) [reply lib/options.json]: done

**Review by alexander-akait — APPROVED** [review]
/cc @snitin315

**Comment by anshumanv** [comment]
Fixing CI

**Review by snitin315 — APPROVED** [review]

**Review by alexander-akait — APPROVED** [review]

**Comment by alexander-akait** [comment]
@anshumanv Please rewrite commit message using conventional commits before merge, other somebody can do invalid release, especial don't not forget to add `BREAKING CHANGE:`

**Comment by anshumanv** [comment]
yep will be mindful 👍

**Pull request merged**